### ISSUE TITLE
Moderise the release instructions

### ISSUE THREAD

**Issue by ennru** [sbt/sbt-site#153]
* Use just a master branch?
* Introduce sbt-dynver?

The Travis secrets for ghpages seem broken, the deploy doesn't work from Travis.

**Comment by jonas** [comment]
I agree that the git-flow based release instructions can be simplified. :+1: to just using a master branch and automatically setting the version in sbt from git using sbt-dynver.

For Travis, I might have revoked the secrets because they were tied to my account.

**Comment by jonas** [comment]
I can look into using deploy keys to update the ghpages branch.

**Comment by ennru** [comment]
Thank you chiming in - good to see you around.
I believe it works well enough to run the ghpages publish from sbt manually.

**Comment by xuwei-k** [comment]
https://github.com/sbt/sbt-site/commit/21649687345e32c683fbed2b9e3e6c43b15a7037#diff-8fd74420ca55aa91d4c53775d2a55a999c405b5a4fafeaf73b7f0ef83e1f3b0cR7-R9

https://github.com/sbt/sbt-site/blob/77711cc5debd3a673acaba5596bfce4eeb4d63a5/project/plugins.sbt#L7
